jedi/evaluate/sys_path.py

```diff
@@ -10,24 +13,78 @@
 from jedi import common
 
 
-def get_sys_path():
-    def check_virtual_env(sys_path):
-        """ Add virtualenv's site-packages to the `sys.path`."""
-        venv = os.getenv('VIRTUAL_ENV')
-        if not venv:
-            return
-        venv = os.path.abspath(venv)
-        p = _get_venv_sitepackages(venv)
-        if p not in sys_path:
-            sys_path.insert(0, p)
-
-        # Add all egg-links from the virtualenv.
+def get_venv_path(venv):
+    """Get sys.path for specified virtual environment."""
+    try:
+        sys_path = _get_venv_path_online(venv)
+    except Exception as e:
```

Comment:
Problem is: Whenever you do something like this, it's really hard to tell what the issue is from the outside. That's why I never do this.